include/swift/Remote/MetadataReader.h

```diff
@@ -2906,9 +2906,11 @@ class MetadataReader {
   template <
       typename T = BuilderType,
       typename std::enable_if_t<
-          !std::is_same<
-              bool,
-              decltype(T::needsToPrecomputeParentGenericContextShapes)>::value,
+          !(std::is_same<
+                const bool,
+                decltype(T::needsToPrecomputeParentGenericContextShapes)>::
+                value &&
```

Comment:
did clang-format put the `value` on a separate line from its `::`?

Reply:
Yes:
```
~/git/swift-project/swift (remote-ast-runtime-type-mismatch) git-clang-format HEAD~1
changed files:
    include/swift/Remote/MetadataReader.h
~/git/swift-project/swift (remote-ast-runtime-type-mismatch) git diff
diff --git a/include/swift/Remote/MetadataReader.h b/include/swift/Remote/MetadataReader.h
index 376135f3afe..c1faf54f89d 100644
--- a/include/swift/Remote/MetadataReader.h
+++ b/include/swift/Remote/MetadataReader.h
@@ -2908,8 +2908,9 @@ private:
       typename std::enable_if_t<
           !(std::is_same<
                 const bool,
-                decltype(T::needsToPrecomputeParentGenericContextShapes)>::value
-                && T::needsToPrecomputeParentGenericContextShapes),
+                decltype(T::needsToPrecomputeParentGenericContextShapes)>::
+                value &&
+            T::needsToPrecomputeParentGenericContextShapes),
           bool> = true>
   BuiltTypeDecl buildNominalTypeDecl(ContextDescriptorRef descriptor) {
     // Build the demangling tree from the context tree.
```

Reply:
that's unfortunate!